src/python/gem5/components/boards/kernel_disk_workload.py

```diff
@@ -139,6 +154,7 @@ def set_kernel_disk_workload(
         kernel: KernelResource,
         disk_image: DiskImageResource,
         bootloader: Optional[BootloaderResource] = None,
+        _disk_device: Optional[str] = None,
```

Comment:
I changed this parameter to disk_device, however, I kept leading underscore in the member name  (self._disk_device), since it isn't a sim obj and it wasn't running when I removed it.